envisage/tests/plugins/orange/plugins.py

```diff
@@ -15,6 +15,3 @@ def get_plugins():
     from .orange_plugin import OrangePlugin
 
     return [OrangePlugin()]
-
-
-#### EOF #######################################################################
```

Comment:
do we want to remove EOF comments from all files instead of fixing some and removing others?

Reply:
Meh. I _do_ want to remove EOF comments from all files, but not in this PR. If we're going to be perfectly clean about this, I should go through and reinstate those comments in this PR, so that they can all be removed together. But that seems like a bit of a waste of time, and I'm not sure I have the energy.

Reply:
I've restored this one.